datahub-graphql-core/src/main/resources/search.graphql

```diff
@@ -150,22 +165,17 @@ input FacetFilterInput {
   field: String!
 
   """
-  Value of the field to filter by (soon to be deprecated)
-  """
-  value: String!
-
```

Comment:
note: this is a breaking change (we are removing the `value` field from `FacetFilterInput`